lib/usd/utils/MergePrimsOptions.cpp

```diff
@@ -0,0 +1,191 @@
+            key.GetText());
+        return def;
+    }
+
+    MergeVerbosity verbosity = MergeVerbosity::None;
+
+    const auto tokens = VtDictionaryGet<std::vector<VtValue>>(options, key);
+    for (const auto& token : tokens) {
+        if (UsdMayaMergeOptionsTokens->None == token)
+            verbosity = verbosity | MergeVerbosity::None;
+        if (UsdMayaMergeOptionsTokens->Same == token)
+            verbosity = verbosity | MergeVerbosity::Same;
+        if (UsdMayaMergeOptionsTokens->Differ == token)
+            verbosity = verbosity | MergeVerbosity::Differ;
+        if (UsdMayaMergeOptionsTokens->Child == token)
+            verbosity = verbosity | MergeVerbosity::Child;
+        if (UsdMayaMergeOptionsTokens->Children == token)
+            verbosity = verbosity | MergeVerbosity::Children;
+        if (UsdMayaMergeOptionsTokens->Failure == token)
+            verbosity = verbosity | MergeVerbosity::Failure;
+        if (UsdMayaMergeOptionsTokens->Default == token)
+            verbosity = verbosity | MergeVerbosity::Default;
+    }
+
+    return verbosity;
+}
+
+/// Extracts a MergeMissing array of tokens at \p key from \p options, or All if it can't
+/// extract.
+MergeMissing parseMissingHandling(
+    const VtDictionary& options,
+    const TfToken&      key,
+    MergeMissing        def = MergeMissing::All)
+{
+    if (!VtDictionaryIsHolding<std::vector<VtValue>>(options, key)) {
+        TF_CODING_ERROR(
+            "Dictionary is missing required key '%s' or key is "
+            "not a vector of tokens",
+            key.GetText());
+        return def;
+    }
+
+    MergeMissing missingHandling = MergeMissing::None;
+
+    const auto tokens = VtDictionaryGet<std::vector<VtValue>>(options, key);
+    for (const auto& token : tokens) {
+        if (UsdMayaMergeOptionsTokens->None == token)
+            missingHandling = missingHandling | MergeMissing::None;
+        if (UsdMayaMergeOptionsTokens->Create == token)
+            missingHandling = missingHandling | MergeMissing::Create;
+        if (UsdMayaMergeOptionsTokens->Preserve == token)
+            missingHandling = missingHandling | MergeMissing::Preserve;
+        if (UsdMayaMergeOptionsTokens->All == token)
+            missingHandling = missingHandling | MergeMissing::All;
+    }
+
+    return missingHandling;
+}
+
+} // namespace
+
+/* static */
+const VtDictionary& MergePrimsOptions::getDefaultDictionary()
+{
+    static VtDictionary   d;
+    static std::once_flag once;
+    std::call_once(once, []() {
+        const auto _token = VtValue(TfToken());
+        const auto _tokenVector = VtValue(std::vector<VtValue>({ _token }));
+
+        d[UsdMayaMergeOptionsTokens->verbosity]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->Default) }));
+        d[UsdMayaMergeOptionsTokens->mergeChildren] = false;
+        d[UsdMayaMergeOptionsTokens->ignoreUpperLayerOpinions] = false;
+        d[UsdMayaMergeOptionsTokens->propertiesHandling]
```

Comment:
Gotta be a way to turn this into a loop...
std::vector<TfToken> mergeOptionTokens{UsdMayaMergeOptionsTokens->propertiesHandling, UsdMayaMergeOptionsTokens->primsHandling, ...};
auto defaultValue = VtValue(std::vector<VtValue>({VtValue(UsdMayaMergeOptionsToken->All)}));
for (const auto& token : mergeOptionTokens) { ... };